node/src/reactor/participating.rs

```diff
@@ -444,44 +496,12 @@ impl Reactor {
         serialized_id: &[u8],
     ) -> Effects<<Self as reactor::Reactor>::Event> {
         match tag {
-            Tag::Deploy => {
-                Self::respond_to_fetch(effect_builder, serialized_id, sender, |deploy_hash| {
-                    self.storage.get_deploy(deploy_hash)
-                })
-            }
-            Tag::Block => {
-                Self::respond_to_fetch(effect_builder, serialized_id, sender, |block_hash| {
-                    self.storage.read_block(&block_hash)
-                })
-            }
-            Tag::BlockAndMetadataByHeight => {
-                Self::respond_to_fetch(effect_builder, serialized_id, sender, |block_height| {
-                    self.storage
-                        .read_block_and_sufficient_finality_signatures_by_height(block_height)
-                })
-            }
-            Tag::GossipedAddress => {
-                warn!("received get request for gossiped-address from {}", sender);
-                Effects::new()
-            }
-            Tag::BlockHeaderByHash => {
-                Self::respond_to_fetch(effect_builder, serialized_id, sender, |block_hash| {
-                    self.storage.read_block_header_by_hash(&block_hash)
-                })
-            }
-            Tag::BlockHeaderAndFinalitySignaturesByHeight => {
-                Self::respond_to_fetch(effect_builder, serialized_id, sender, |block_height| {
-                    self.storage
-                        .read_block_header_and_sufficient_finality_signatures_by_height(
-                            block_height,
-                        )
-                })
-            }
             Tag::Trie => {
                 Self::respond_to_fetch(effect_builder, serialized_id, sender, |trie_key| {
                     self.contract_runtime.get_trie(trie_key)
                 })
             }
+            _ => todo!("already handled"),
```

Comment:
Should this method be removed then? Or called `handle_trie_request` and not have a `tag` argument?

Reply:
This will cause panic if hit.

Reply:
True. It should in theory be `unreachable!`, but I will replace it with a `warn!` instead, just in case.